### DISCUSSION TITLE
Visual Significances

### DISCUSSION THREAD

**Discussion by levone1** [jarun/nnn#1784] [Q&A] [answered]
Maybe there's not really anything to this, but I just can't get past this feeling that I'm missing something...

Over time, I've been able to realize, by usage, the significance of certain visual elements in the file lists, like when a text file is empty, it shows it's name in red, then changes color when it contains anything.  I haven't been able to find anywhere in the documentation where this is mentioned, (note - it very well may be there).  There are other visual things I wonder about over time, some of which I've figured out, and some not, (like sometimes the file properties are in a color-inverted "block", and sometimes not, etc.).  Is there anywhere in the docs that all these visual cues are defined?

Thanks

**Comment by N-R-K** [comment, marked as answer]
> Is there anywhere in the docs that all these visual cues are defined?

Can't remember anything off the top of my head.

> like sometimes the file properties are in a color-inverted "block"

I think that's when a file is "recently modified/created". I personally don't like that feature and thus have it patched out in my local build.

**Reply by 7ocb** [reply]
The "inverted colors" feature described [here](https://github.com/jarun/nnn/wiki/Usage): 

> Timestamps for entries created/modified within 5 minutes are shown in reverse.
